backend/tests/services/rag/storage/test_milvus_backend.py

```
@@ -0,0 +1,432 @@
+                "enable_sparse"
+            )
+
+    def test_create_vector_store_for_hybrid_mode(self, milvus_backend):
+        """Test creating vector store for hybrid search."""
+        with patch(
+            "app.services.rag.storage.milvus_backend.MilvusVectorStore"
+        ) as mock_store_class:
+            mock_store = MagicMock()
+            mock_store_class.return_value = mock_store
+
+            result = milvus_backend.create_vector_store(
```

Ruff (0.15.0)

[error] Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)